pages/static/src/scss/helpers/_css-helpers.scss

```diff
@@ -1154,6 +1154,23 @@ a.white-text {
   }
 }
 
+
+.our-dream-bg {
```

Comment:
Seems odd to me that this is in a file called css-helpers, but we haven't really figured out our standard yet.

Reply:
Agreed, when we solidify CSS conventions we'll address.